README.md

```diff
@@ -1,3 +1,4 @@
+[![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=azure_nomad_repo)
```

Comment:
We'll eventually see a whole long list of these in Google Analytics, so perhaps we should use `repo_azure_nomad` so that all our referrals from repos will be alphabetically listed?